CHANGELOG.md

```diff
@@ -6,6 +6,32 @@
 
 ### Fixes
 
+## v1.29.0 (2023-04-10)
+
+### Enhancements
+
+- Batch Download of traces ([@Katarzyna-B](https://github.com/Katarzyna-B) in [#1274](https://github.com/jaegertracing/jaeger-ui/pull/1274))
```

Comment:
I just merged a fix for the file format, but as far as changelog I think it's good as is